api/types/client.go

```diff
@@ -126,21 +126,21 @@ type ImageBuildOptions struct {
 	Remove         bool
 	ForceRemove    bool
 	PullParent     bool
-	Isolation      string
-	CPUSetCPUs     string
-	CPUSetMems     string
-	CPUShares      int64
-	CPUQuota       int64
-	CPUPeriod      int64
-	Memory         int64
-	MemorySwap     int64
-	CgroupParent   string
-	ShmSize        string
-	Dockerfile     string
-	Ulimits        []*units.Ulimit
-	BuildArgs      []string
-	AuthConfigs    map[string]AuthConfig
-	Context        io.Reader
+	container.IsolationLevel
```

Comment:
Thanks for the review. I embedded it intentionally, since I was going to rename it to "IsolationLevel" and figured that it might be easier to embed.